File: solutions/cpp/easy/classes-objects/solution-approach1.cpp
// ──────────────────────────────────────────────────
// Link        https://www.hackerrank.com/challenges/classes-objects/problem?isFullScreen=true
// Problem     Classes and Objects
// Difficulty  Easy
// Subdomain   Classes
// Platform    HackerRank
// Language    cpp
// Status      Accepted
// Submitted   2026-09-01, 08:39 p.m.
// ──────────────────────────────────────────────────


// Write your Student class here
class Student {
    public:
        int scores[5];
        int sum = 0;
        void input() {
            for (int i = 0; i < 5; i++) {
                cin >> scores[i];
            }
        }
        int calculateTotalScore() {
            sum = 0;
            for (int i = 0; i < 5; i++) {
                sum += scores[i];
            }
            return sum;
        }
};

File: solutions/python3/medium/write-a-function/solution-approach1.py
# ──────────────────────────────────────────────────
# Link        https://www.hackerrank.com/challenges/write-a-function/problem?isFullScreen=true
# Problem     Write a function
# Difficulty  Medium
# Subdomain   Introduction
# Platform    HackerRank
# Language    python3
# Status      Accepted
# Submitted   2026-09-21, 07:18 a.m.
# ──────────────────────────────────────────────────

def is_leap(year):
    leap = False
    if (year==2100):
        return leap
    # Write your logic here
    if(year%4==0):
        leap=True
        return leap
    return leap

